specification/2.0/schema/extras.schema.json

```diff
@@ -1,5 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema",
     "title": "Extras",
-    "description": "Application-specific data."
+    "description": "Application-specific data.",
+    "gltf_detailedDescription": "Application-specific data. **Implementation Note:** Although extras may have any type, it is common for applications to store and access custom data as key/value pairs. As best practice, extras should be an Object rather than a primitive value for best portability."
```

Comment:
Does `**Implementation Note:**` decoration look as expected in the generated Markdown?

Reply:
No, and worse, it sprinkles itself all over the reference section.

Would it be OK if I add a new type here called `gltf_sectionDescription` ?  The "detailedDescription" one is only ever used on properties, not at the top level of a reference section like this.  I have a local 2-line change to wetzel that will kick it out in the correct place.

Reply:
Here's the proposed change to wetzel: https://github.com/CesiumGS/wetzel/pull/42

Reply:
Nothing prevents us from adding custom properties, so let's do it.

Reply:
Done.